wagtail_newsletter/migrations/0001_initial.py

```diff
@@ -27,4 +35,90 @@ class Migration(migrations.Migration):
             ],
             options={"verbose_name_plural": "Newsletter recipients"},
         ),
+        migrations.CreateModel(
```

Comment:
Yep, I'm changing the existing Django migration. As the package hasn't been released yet, I think it's worth keeping the number of migrations to a minimum.